components/vf-stack/vf-stack--500.njk

```diff
@@ -0,0 +1,18 @@
+{% if context %}
```

Comment:
Shouldn't one vf-stack.njk template be enough, or is there something I'm missing?

Reply:
It's in the hope that people using `njk` in their sites make use of nunjucks' `block` to save people writing more HTML (that could well change) than they have to.

Reply:
I'd think one template should still be enough? There's probably some technical thing with `block` that I'm not missing, we can chat about it sometime. Certainly not a reason to hold back the PR.

Reply:
I'll try and remember why I had to go this way - rather than rely on using blocks in fractal -- I think it was because of their use of `@` to call things -- which didn't work in 11ty -- but would do if we kept a list of key/value pairs like we do with includes.

Reply:
I'll merge this in as it works 'as is' - and look to check how the alternative way works with fractal and 11ty